Essentials/src/messages.properties

```diff
@@ -178,7 +178,7 @@ helpOp=\u00a74[HelpOp]\u00a7r \u00a76{0}\:\u00a7r {1}
 helpPlugin=\u00a74{0}\u00a7r\: Plugin Help\: /help {1}
 holdBook=\u00a74You are not holding a writable book.
 holdFirework=\u00a74You must be holding a firework to add effects.
-holdPotion=\u00a74You must be holding a potion to apply effects to it.
+holdPotion=\u00a74You must be holding a potion or an arrow to apply effects to it.
```

Comment:
This message shouldn't be changed as we still support 1.8.x and this would be wrong in 1.8.x

Reply:
I'd recommend adding a separate `holdPotionOrArrow` message for appropriate versions.